app/controllers/reports_controller.rb

```diff
@@ -31,7 +37,10 @@ def report_params
     development_params = (1..ReportForm::DEVELOPMENT_OBJECTIVES).map do |n|
       "development_#{n}"
```

Comment:
So I think wouldn't be able to use FormBuilder, I'd need to manually populate the form. Not sure if that would make it easier at this point. Let's look at it again, when it gets messier.